app/src/main/res/layout/feeditemlist_header.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
+    android:background="#5000"
```

Comment:
can this value be read from somewhere? or are we sure that it is the same color across every version of android, manufacturer variants and roms?

Reply:
This is just a fixed ARGB color. Should be the same on every device. If you want, I can extract it to a `@color` resource or use the "full" format (`#55000000`).

I have chosen the color arbitrarily. Just wanted to darken the background a bit (regardless of the theme) to prevent feed images with alpha channel from looking strange (see screenshot).

Reply:
yes, extract to `colors.xml` please